### PR TITLE
DOCSP-41950: Landing page

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-php-library/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-41950
Staging - https://deploy-preview-150--docs-php-library.netlify.app/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/index.txt [hunk 1]
```diff
@@ -18,68 +18,104 @@ MongoDB PHP Library
...
+-----------
+
+Learn how to install the library and extension, establish a connection to MongoDB,
+and begin working with data in the :ref:`php-get-started` tutorial.
+
+Connect to MongoDB
+------------------
+
+Learn how to create and configure a connection to a MongoDB deployment
+in the :ref:`php-connect` section.
```
Comment (norareidy) [inline source/index.txt:51]: Note to reviewer: link broken until #147 is merged

File: snooty.toml [hunk 2]
```diff
@@ -22,8 +22,10 @@ toc_landing_pages = [
...
     "/reference/class/MongoDBModelIndexInfo",
     "/get-started",
+    "/databases-collections",
     "/write",
-    "/indexes"
+    "/indexes",
+    "/data-formats"
 ]
 
 sharedinclude_root =  "https://raw.githubusercontent.com/10gen/docs-shared/main/"
```
Comment (rustagir) [inline snooty.toml:31]: S: should we rename "driver-short"? Line 37

File: source/index.txt [hunk 3]
```diff
@@ -18,68 +18,104 @@ MongoDB PHP Library
...
+----------
 
-- :manual:`Introduction to MongoDB </introduction>`
+Learn about new features and changes in each version in the :ref:`<php-lib-whats-new>`
+section.
 
-- :manual:`Databases and Collections </core/databases-and-collections>`
+.. TODO:
+ Upgrade {+driver-short+} Versions
+ ---------------------------------
```
Comment (rustagir) [inline source/index.txt:113]: S: add newline under heading

File: source/index.txt [hunk 4]
```diff
@@ -18,68 +18,104 @@ MongoDB PHP Library
...
+ Learn what changes you might need to make to your application to upgrade driver versions
```
Comment (rustagir) [inline source/index.txt:114]:
```suggestion
 Learn what changes you must make to your application to upgrade driver versions
```

**Review by rustagir — APPROVED** [review]
lgtm with a few small suggestions!

**Pull request merged**